third_party/zlib.BUILD

```diff
@@ -1,16 +1,65 @@
-# From https://github.com/bazelbuild/bazel/blob/master/third_party/zlib/BUILD
+# Originally from https://github.com/protocolbuffers/protobuf/blob/2514f0bd7da7e2af1bed4c5d1b84f031c4d12c10/third_party/zlib.BUILD
```

Comment:
out of curiosity, why did this zlib.BUILD file change?

Reply:
We had two versions committed, one in sorbet_llvm and one in sorbet. I standardized on the one that built successfully with the sanitizer on, though that might not be necessary now that we're running the sorbet_llvm tests completely separately.